lib/helpers/git_helper.rb

```diff
@@ -0,0 +1,39 @@
+require 'json'
+require 'git'
+require 'fileutils'
+
+def get_config (config_file)
+  json_data = []
+  begin
+    json_file = File.read(config_file)
+    json_data = JSON.parse(json_file)
+  rescue Exception => e
+    puts e.message
+    puts e.backtrace.inspect
+    json_data = nil
+  end
+  json_data
+end
+
+def clone_repo (repo, branch)
+  if File.directory?('blog content')
```

Comment:
Does it matter if the file is a directory? If it exists as a file it will still stop the ability to clone.